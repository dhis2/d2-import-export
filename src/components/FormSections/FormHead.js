import React from 'react'
import styles from './FormHead.module.css'

export const FormHead = ({ icon, label }) => (
    <div className={styles.container}>
        <span className={styles.icon}>{icon}</span>
        <span className={styles.label}>{label}</span>
    </div>
)
