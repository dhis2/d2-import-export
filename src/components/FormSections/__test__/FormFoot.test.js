import React from 'react'
import renderer from 'react-test-renderer'
import { FormHead } from '../FormHead'

describe('Input component - FormHead', () => {
    it('should render correctly', () => {
        const file = renderer
            .create(<FormHead icon={<span />} label="Form head" />)
            .toJSON()

        expect(file).toMatchSnapshot()
    })
})
