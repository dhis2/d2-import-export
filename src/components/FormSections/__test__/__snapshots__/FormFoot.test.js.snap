// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Input component - FormHead should render correctly 1`] = `
<div
  className="container"
>
  <span
    className="icon"
  >
    <span />
  </span>
  <span
    className="label"
  >
    Form head
  </span>
</div>
`;
